### PR TITLE
Theme provider

### LINKED ISSUE #4: DEV: Set up a ThemeProvider
The Styled Components ThemeProvider is a component which will essentially pass down any styling-oriented props (colours, fonts, etc) into any Styled Component. Using a ThemeProvider allows all of the styling-oriented values (background colours, font sizes, etc) to be exist in one location to be easily updated if needed.

This [guide](https://dev.to/aromanarguello/how-to-use-themes-in-styled-components-49h) is a good reference to set up theming in our app.

Please follow the guide and create a working Theme that `App.tsx` will be import and use.

The `App` functional component should return JSX such that everything within the App is encapsulated by the `ThemeProvider` component.

That means the App return value should look like:
```js
const App = ()=> {
// stuff
  return (
    <Theme>
      {/** Other stuff */}
    </Theme>
  );
}
```

### PR DESCRIPTION
**Pull request by meerabal**
fixes #4

### CODE REVIEW

File: src/content/fonts/fonts.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import PoppinsEOT from './poppins-v15-latin-regular.eot';
+import PoppinsWOFF2 from './poppins-v15-latin-regular.woff2';
+import PoppinsWOFF from './poppins-v15-latin-regular.woff';
+import PoppinsSVG from './poppins-v15-latin-regular.svg';
+
+import PoppinsTTF from './poppins-v15-latin-regular.ttf'; //'./Poppins-SemiBold.ttf';
```
Comment (deenhaque) [inline src/content/fonts/fonts.ts]: Kill the comment here?

File: src/Theme.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    heading: "Poppins, sans-serif",
+    content: "Source Sans Pro, sans-serif"
+  },
+  fontSizes: {
+    small: "1em",
+    medium: "2em",
+    large: "3em"
+  },
+  viewport: {
+    mobile: "(min-width: 425px)",
```
Comment (deenhaque) [inline src/Theme.tsx]: I think this should be 320px since no styling will apply on an iPhone X w/ this min-width here

File: src/App.tsx [hunk 3]
```diff
@@ -21,7 +25,9 @@ const links = [
 const App = () => {
   console.log(links.map((link) => Object.keys(link)[0]));
```
Comment (deenhaque) [inline src/App.tsx]: I think we can kill this line

**Review by deenhaque — COMMENTED** [review]
Looks great! Left some small feedback :)

File: src/content/fonts/fonts.ts (hunk 1 above)
Comment (Seneli) [reply src/content/fonts/fonts.ts]: are we switching to a different font?

**Comment by Seneli** [comment]
I think it looks great too. I just had a quick question about the fonts since I saw one was commented out.

**Pull request merged**